minesweeper/domain/models/Cell.model.ts

```diff
@@ -1,5 +1,16 @@
 import { IPosition } from '@minesweeper/domain/commons.type';
 
 export interface CellModel extends IPosition {
-  exposed: boolean;
+  /**
+   * represent internal state of cell
+   *
+   * if is "undefined" it means that the cell is not exposed and the user doesn't know if this cell is a bomb or not
+   */
+  isBomb?: boolean;
```

Comment:
I changed idea of "state" on cell Model for property "isBomb", i think this way is more intuitive

Reply:
I prefer the word hasBomb instead of isBomb, I think it is more clear

Reply:
Okaaa